cloudbuild.yaml

```diff
@@ -1,10 +1,12 @@
 steps:
 
 - name: 'gcr.io/cloud-builders/gcloud'
-  args: ['builds','submit','--config=ci/cloudbuild.yaml','--substitutions','TAG_NAME=$TAG_NAME,COMMIT_SHA=$COMMIT_SHA,_BUILD_TYPE=']
+  id: "standard"
+  args: ['builds','submit','--region=us-central1','--config=ci/cloudbuild.yaml','--substitutions','TAG_NAME=$TAG_NAME,COMMIT_SHA=$COMMIT_SHA,_BUILD_TYPE=']
```

Comment:
Are we interested in adding either the `repo_envoy-gloo` or `main-job` tags on these jobs? Should we include the build type or any other information as a tag?

Reply:
`repo_envoy-gloo` is added by build bot [here](https://github.com/solo-io/build-bot/blob/master/pkg/internal/build/build.go#L115).

I could see value in adding `main-job` but its omission is also good enough for me. Happy to add it though

Reply:
given that there are only two job types, `main-job` doesn't seem necessary. resolving the thread.